pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>edu.illinois.cs</groupId>
     <artifactId>testrunner</artifactId>
-    <version>1.1-SNAPSHOT</version>
+    <version>1.2-SNAPSHOT</version>
```

Comment:
I can see why it would be good to update version number for big change like adding JUnit 5, but we have yet to release 1.1 (without SNAPSHOT). We should either keep this as 1.1-SNAPSHOT and release this as part of 1.1, or release 1.1 before these changes and start 1.2-SNAPSHOT with this JUnit 5 change.

Reply:
Oh! I thought 1.1 have been released. And Done.